sync/p2p_messages.md

```diff
@@ -12,7 +12,7 @@ Noise (and the fragmentation) handle message size we need no length field. The
 payload is a byte array, and messages are either fixed binary data or encoded
 using [RLP](https://github.com/ethereum/wiki/wiki/RLP).
 
-There are currently 13 different P2P messages implemented in
+There are currently 15 different P2P messages implemented in
```

Comment:
Maybe removing the number for easing maintenance ? E.g. "The P2P messages currently implemented in [epoch](https://github.com/aeternity/epoch/blob/master/apps/aecore/src/aec_peer_messages.erl) are:" ?